README.md

```diff
@@ -1 +1,26 @@
 # MaximumGeneratorProfitTCRDD
+
```

Comment:
The README needs a bit of reformatting. Ideally each sentence would be on its own line.